Add Replicate demo and API

chenxwh opened the pull request:
Hey @rentainhe! 👋 

Great work and fun project!

This pull request makes it possible to run RAM-Grounded-SAM & SAM-HQ Replicate (https://replicate.com/cjwbw/ram-grounded-sam) and via API:

```
import replicate
model = replicate.models.get('cjwbw/ram-grounded-sam')
model.predict(image=...)
```
We would also like to invite you to transfer the ownership to `IDEA-Research` so you could easily modify the model and push changes :)

rentainhe commented:
Thank you so much for providing such a nice platform for us to show the demo! We're very likely to highlight this nice demo playground

> We would also like to invite you to transfer the ownership to IDEA-Research so you could easily modify the model and push changes :)

I was wondering what should I do to update the demo~, maybe we can inform you if there're any updates of our demo~

chenxwh commented:
Thank you for the feedback! I have now added GroundingDINO  the citation and updated the page under `IDEA-Research`.

To modify the page, simply sign-in to Replicate with your Github account [here](https://replicate.com/signin?next=/), and let me know - I will add you to the `IDEA-Research` organisation. When signed-in you can easily modify the README, example pages etc on the browser. 

For further updating the model, here is the documentation of [Cog](https://github.com/replicate/cog) that powers Replicate on how to push and modify models, but if it is complicated we can always help you to update and maintain the page!

rentainhe commented:
> Thank you for the feedback! I have now added GroundingDINO the citation and updated the page under `IDEA-Research`.
> 
> To modify the page, simply sign-in to Replicate with your Github account [here](https://replicate.com/signin?next=/), and let me know - I will add you to the `IDEA-Research` organisation. When signed-in you can easily modify the README, example pages etc on the browser.
> 
> For further updating the model, here is the documentation of [Cog](https://github.com/replicate/cog) that powers Replicate on how to push and modify models, but if it is complicated we can always help you to update and maintain the page!

Got it! Thank you so much! I will send you en email after I registered Replicate~
